### ISSUE TITLE
Remove manual prompt based signing path from forc-client

### ISSUE THREAD

**Issue by kayagokalp** [FuelLabs/sway#6071]
From couple of years ago, at the time we first introduced forc-deploy, we did not have any way of signing transactions other than manual copy pasting in between multiple terminals. This was resolved with forc-wallet integration but the old path is still in the code. We should remove it as it is pretty outdated, although working probably confusing for someone new into the codebase and makes some new stuff little bit harder than it needs to be like #6069 

Basically remove:
https://github.com/FuelLabs/sway/blob/f342f52381c36730094106d5b9c8d7e1e960c1d4/forc-plugins/forc-client/src/util/tx.rs#L41-L56

and places calling into these functions.
